modules/apps/dynamic-data-mapping/dynamic-data-mapping-service/src/main/java/com/liferay/dynamic/data/mapping/internal/upgrade/v5_3_2/TemplateBrowserSnifferUpgradeProcess.java

```diff
@@ -0,0 +1,40 @@
+/**
```

Comment:
@alberttgomes, can we rephrase the commit message to:

Create upgrade process to remove declared browser sniffer variables